charts/nextcloud/templates/_helpers.tpl

```diff
@@ -246,11 +246,13 @@ Create volume mounts for the nextcloud container as well as the cron sidecar con
 - name: nextcloud-main
   mountPath: /var/www/html
   subPath: {{ ternary "html" (printf "%s/html" .Values.nextcloud.persistence.subPath) (empty .Values.nextcloud.persistence.subPath) }}
-{{- if and .Values.persistence.nextcloudData.enabled .Values.persistence.enabled }}
+{{- if .Values.persistence.nextcloudData.enabled }}
 - name: nextcloud-data
   mountPath: {{ .Values.nextcloud.datadir }}
-  subPath: {{ ternary "data" (printf "%s/data" .Values.persistence.nextcloudData.subPath) (empty .Values.persistence.nextcloudData.subPath) }}
-{{- else }}
+{{- if not (empty .Values.persistence.nextcloudData.subPath) }}
+  subPath: {{ .Values.persistence.nextcloudData.subPath }}
+{{- end }}
```

Comment:
This is a breaking change that deserves a bump of the major version

Reply:
Oh I mixed up minor and major, fixed.

I do not really like adding values but am not defensive about it, if you would prefer that.